releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -96,7 +96,7 @@ public void testGenerateOnlyBug() throws Exception {
             createNotes(null, null, createAllNotes(), null, null), createBaseCliOptions()
                 .setGenerateAll(true).build());
 
-        Assert.assertEquals("no errors", 0, errors.size());
+        Assert.assertEquals("one error", 1, errors.size());
```

Comment:
I checked both tests that I changed and we have to expect an error with this configuration. The other solution is to make `createBaseCliOptions` accept an optional parameter `releaseNumber` so we can change from default `1.0.0` to two-number one - `1.0`, for instance.